lib/index.js

```diff
@@ -20,13 +20,48 @@ function throttlingPlugin (octokit) {
     triggersNotificationLimiter: new Bottleneck({
       maxConcurrent: 1,
       minTime: 3000
-    })
-  }
+    }),
+    retryLimiter: new Bottleneck()
+  }, octokitOptions.throttle)
+
+  const events = {}
+  const emitter = new Bottleneck.Events(events)
+  events.on('abuse-limit', state.onAbuseLimit)
+  events.on('rate-limit', state.onRateLimit)
```

Comment:
We should throw an error if `octokitOptions.onRateLimit / `octokitOptions.onAbuseLimit` is not set